src/stuf/stuf_bg/tests/test_client.py

```diff
@@ -64,7 +64,7 @@ def test_get_address(self, _mock):
                 ),
             )
             self.client.get_values_for_attributes(
-                "999992314", list(FieldChoices.values.keys())
+                "999992314", [c.value for c in FieldChoices]
```

Comment:
Could this not be:
```suggestion
                "999992314", FieldChoices.values
```
?